docs/HyperIndex/Guides/cli-commands.md

```diff
@@ -317,3 +318,13 @@ Start the indexer without any automatic codegen
 ###### **Options:**
 
 * `-r`, `--restart` — Clear your database and restart indexing from scratch
+
+
+
+<hr/>
+
+<small><i>
+    This document was generated automatically by
+    <a href="https://crates.io/crates/clap-markdown"><code>clap-markdown</code></a>.
+</i></small>
+
```

Comment:
Should be removed

Reply:
Yeah, just reran the script to get the latest update from indexer